app/screens/home/DetailsScreen.tsx

```diff
@@ -514,6 +535,10 @@ console.log(videoIsFocused,"..videoIsFocused");
               fromScreen === "Home" ?
               <>
               <TrackMilestoneView currentSelectedChildId={currentSelectedChildId}/>
+              <FlexCol style={{ backgroundColor: newBackgroundColor }}>
+                    <RelatedArticles relatedArticles={detailDataToUse?.related_articles} category={detailDataToUse?.category} fromScreen={fromScreen} currentId={detailDataToUse?.id} headerColor={newHeaderColor} backgroundColor={newBackgroundColor} listCategoryArray={listCategoryArray} navigation={navigation} currentSelectedChildId={currentSelectedChildId} />
+                    <RelatedVideoArticles relatedArticles={detailDataToUse?.related_video_articles ? detailDataToUse?.related_video_articles : []} category={detailDataToUse?.category} fromScreen={fromScreen} currentId={detailDataToUse?.id} headerColor={newHeaderColor} backgroundColor={newBackgroundColor} listCategoryArray={listCategoryArray} navigation={navigation} currentSelectedChildId={currentSelectedChildId} />
+              </FlexCol>
```

Comment:
Can we combine like: 
            {
              fromScreen === "Home" || fromScreen === "ChildDevelopment" ?
                <>
                  <TrackMilestoneView currentSelectedChildId={currentSelectedChildId} />
                  <FlexCol style={{ backgroundColor: newBackgroundColor }}>
                    ...
                  </FlexCol>
                </> : null
            }

Reply:
done